src/controllers/scripting/controllerscriptenginebase.cpp

```diff
@@ -5,7 +5,14 @@
 #include "controllers/controller.h"
```

Comment:
I think it would be best to get at least one more review from someone who has been working on the controller code recently

Reply:
Perhaps @JoergAtGithub can help with this?

Reply:
Perhaps @JoergAtGithub or @Swiftb0y could help with this?